chrome/content/zotero/tools/csledit.xul

```diff
@@ -58,13 +58,13 @@
 					<menuitem label="near-note" value="4"/>
 				</menupopup>
 			</menulist>
-			<menulist id="locale-menu" oncommand="Zotero_CSL_Editor.refresh()"/>
-			<menulist id="zotero-csl-list" style="min-height: 1.6em; min-width: 100px" initialized="false" flex="1" oncommand="Zotero_CSL_Editor.loadCSL(this.selectedItem.value)"/>
+			<menulist id="locale-menu" oncommand="Zotero.Prefs.set('export.lastLocale', this.value); Zotero_CSL_Editor.refresh()"/>
```

Comment:
Yeah, I wasn't sure if people would expect this pref to update when using the CSL Edit pane, so I didn't hook it up before. Can't hurt, I guess.